src/Slices/Orders/UI/Page.tsx

```diff
@@ -39,40 +44,49 @@ export const Page: React.FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [sort.order]);
 
-  return (
-    <PageContainer pageTitle={words("orders.title")}>
-      <TableControls
-        paginationWidget={
-          <OldPaginationWidget
-            data={data}
-            pageSize={pageSize}
-            setPageSize={setPageSize}
-            setCurrentPage={setCurrentPage}
+  if (isError) {
+    return (
+      <PageContainer pageTitle={words("orders.title")}>
+        <ErrorView ariaLabel="OrdersView-Error" retry={refetch} message={error.message} />;
+      </PageContainer>
+    );
+  }
+
+  if (isSuccess) {
+    return (
+      <PageContainer pageTitle={words("orders.title")}>
+        <TableControls
+          paginationWidget={
+            <PaginationWidget
+              data={data}
+              pageSize={pageSize}
+              setPageSize={setPageSize}
+              setCurrentPage={setCurrentPage}
+            />
+          }
+        />
+        {data.data.length <= 0 || disabledOrderView ? (
+          <EmptyView
+            message={disabledOrderView ? words("orders.disabled") : words("orders.table.empty")}
+            aria-label="OrdersView-Empty"
           />
-        }
-      />
-      <RemoteDataView
-        data={data}
-        label="OrdersView"
-        retry={retry}
-        SuccessView={(orders) =>
-          orders.data.length <= 0 || disabledOrderView ? (
-            <EmptyView
-              message={disabledOrderView ? words("orders.disabled") : words("orders.table.empty")}
-              aria-label="OrdersView-Empty"
+        ) : (
+          <div aria-label="OrdersView-Success">
+            <OrdersTable
+              rows={data.data}
+              tablePresenter={new OrdersTablePresenter()}
+              sort={sort}
+              setSort={setSort}
             />
-          ) : (
-            <div aria-label="OrdersView-Success">
-              <OrdersTable
-                rows={orders.data}
-                tablePresenter={new OrdersTablePresenter()}
-                sort={sort}
-                setSort={setSort}
-              />
-            </div>
-          )
-        }
-      />
+          </div>
+        )}
+      </PageContainer>
+    );
+  }
+
+  return (
+    <PageContainer pageTitle={words("orders.title")}>
+      <LoadingView ariaLabel="OrdersView-Loading" />{" "}
```

Comment:
{" "} is that needed?

Reply:
Nope, auto-linting on save preserved trailing space and I missed that